src/garage/tf/samplers/ray_batched_sampler.py

```diff
@@ -0,0 +1,71 @@
+"""Ray Sampler, for tensorflow algorithms.
+
+Currently the same as garage.samplers.RaySampler but includes
+support for Tensorflow sessions
+"""
+import ray
+import tensorflow as tf
+
+from garage.sampler import RaySampler, SamplerWorker
+
+
+class RaySamplerTF(RaySampler):
+    """Ray Sampler, for tensorflow algorithms.
+
+    Currently the same as garage.samplers.RaySampler
+
+    Args:
+        - Same as garage.samplers.RaySampler
+    """
+
+    def __init__(self,
+                 algo,
+                 env,
+                 seed,
+                 should_render=False,
+                 num_processors=None):
+        super().__init__(
+            algo,
+            env,
+            seed,
+            should_render=False,
+            num_processors=None,
+            sampler_worker_cls=SamplerWorkerTF)
+
+    def shutdown(self):
+        """Shuts down the worker."""
+        temp = []
+        for worker in self._all_workers.values():
+            temp.append(worker.shutdown.remote())
+        ray.get(temp)
+        ray.shutdown()
+
+
+class SamplerWorkerTF(SamplerWorker):
+    """Sampler Worker for tensorflow on policy algorithms.
+
+    - Same as garage.samplers.SamplerWorker, except it
+    initializes a tensorflow session, because each worker
+    is in a separate process.
+    """
+
+    def __init__(self,
+                 worker_id,
+                 env,
+                 agent,
+                 seed,
+                 max_path_length,
+                 should_render=False):
```

Comment:
I don't think we should render environment in a sampler. We should just leave this to a plotter.

Reply:
I can change that, but I think @krzentner was saying that it might be useful to render one of the sampler workers for debugging. KR can you comment on this?

Reply:
Yeah, it's sometimes pretty useful to be able to render the environment in the sampler for debugging purposes. I know the conventional way of watching observing the policy is to use a separate script, but that doesn't always give the same information.

As long as doing this isn't a huge burden, I would prefer we keep it.